sound/soc/sof/pm.c

```diff
@@ -9,35 +9,316 @@
  */
 
```

Comment:
/data/pbossart/ktest/sof-dev/sound/soc/sof/pm.c: In function ‘sof_suspend_streams’:
/data/pbossart/ktest/sof-dev/sound/soc/sof/pm.c:177:9: error: ‘struct snd_sof_pcm’ has no member named ‘restore_stream’
     spcm->restore_stream[dir] = 1;
         ^~
/data/pbossart/ktest/sof-dev/sound/soc/sof/pm.c:194:9: error: ‘struct snd_sof_pcm’ has no member named ‘restore_stream’
     spcm->restore_stream[dir] = 1;
         ^~